#160617658 Change project from Django1 to Django2

kimbugp opened the pull request:
#### What does this PR do?
- Change project from Django-1.10 to Django-2.1

#### Description of Task to be completed?
- Replace Django1 with Django2 in requirements file
- Add JWTAUthetication class in backend.py
- Change url patterns to match Django2 syntax
- Update docstrings to match to Django2.1 documentation
- Add empty JWTAuthentication class to backend.py
- Correct django imports to match Django-2.1
#### How should this be manually tested?
- Install requirements file ```pip install -r requirements.txt```
- Make migrations ```python manage.py makemigrations```
- Migrate them ```python manage.py migrate```
- This can be tested by running the command to start Django server ```python manage.py runserver```
#### What are the relevant pivotal tracker stories?
- ```#160617658```

kcharles52 commented on authors/urls.py:
```diff
@@ -1,24 +1,24 @@
 
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/1.11/topics/http/urls/
+TThe `urlpatterns` list routes URLs to views. For more information please see:
```
Thanks for work well done. Please take note that there is a typo ` TThe `

zaabu reviewed (commented):
Good work @kimbugp . I think we should move the SECRET_KEY to another file because exposing it here is a security risk.

kimbugp committed:
chore(upgrade_app):upgrade app from django 1.10 to django 2.1

- Change codebse from Django 1.10 to Django 2.1

[Delivers #160617658]

chore(upgrade-app):Update requirements file

- Add updated modules required to requirements.txt

[Delivers #160617658]

chore(upgrade-app):Update docstrings

-Update docstrings documentation to match Django-2.1

[Delivers #160617658]

chore(upgrade-app):Update settings file

-Update docstrings documentation to match Django-2.1
-Change urlpattern from url to django-2.1 path
-Correct imports to match django-2.1 import paths

[Delivers #160617658]

chore(upgrade-app):Update docstrings in wsgi.py file

-Update docstrings documentation to match Django-2.1

[Delivers #160617658]

chore(upgrade-app):Add JWTAuthetication clas

-Add empty jwtauthentication class to backend.py

[Delivers #160617658]

chore(upgrade-app):change url to path

-Change urlpatterns from url to django 2.1 path
-Correct import to django 2.1

[Delivers #160617658]
